packages/@o3r/core/schematics/sdk/index.ts

```diff
@@ -28,6 +28,15 @@ export function generateSdk(options: NgGenerateSdkSchema): Rule {
     /** Path to the folder where generate the new SDK */
     const targetPath = path.posix.join(options.path || defaultRoot, cleanName);
 
+    const addModuleSpecificFiles = () => mergeWith(apply(url('./templates'), [
+      template({
+        ...options,
+        rootRelativePath: path.relative(targetPath, '.' + tree.root.path)
```

Comment:
```suggestion
        rootRelativePath: path.relative(targetPath, tree.root.path.replace(/^\//, './')
```
Would be safer in case it does not start with `/`.